apps/api/src/campaign-file/campaign-file.controller.ts

```diff
@@ -28,9 +28,11 @@ export class CampaignFileController {
   @UseInterceptors(FilesInterceptor('file'))
   async create(
     @Param('campaign_id') campaignId: string,
+    @Body() body: { filesRole: string },
```

Comment:
@dplamenov Can you please extract this var in dedicated DTO object

In such way you can filter and validate the input prior hitting the controller

Reply:
@kachar, I don't know how to validate `filesRole` because it is a string. Can you provide more details? Please.